latest_cov_build.py

```diff
@@ -22,12 +22,13 @@
 # Download artifacts
 for name in ['target.tar.bz2', 'target.code-coverage-gcno.zip', 'chrome-map.json']:
     url = queue.buildUrl('getLatestArtifact', taskId, 'public/build/{}'.format(name))
-    urlretrieve(url, name)
+    filename = os.path.join('tools', name)
+    urlretrieve(url, filename)
 
 # Extracting coverage build artifact
-with tarfile.open('target.tar.bz2', 'r:bz2') as tar:
+with tarfile.open('tools/target.tar.bz2', 'r:bz2') as tar:
     tar.extractall()
```

Comment:
Isn't extractall going to extract in the current working directory? Perhaps you need to add path='tools' here.

Reply:
Seems that I've forgotten to save this change in the local file. Updated now. :)